docs/data/date-pickers/accessibility/accessibility.md

```diff
@@ -0,0 +1,93 @@
+
+|                                                                  Keys | Description                                  |
+| --------------------------------------------------------------------: | :------------------------------------------- |
+| <kbd class="key">Arrow Left</kbd>, <kbd class="key">Arrow Right</kbd> | Moves focus among date/time sections         |
+|                                       <kbd class="key">Arrow Up</kbd> | Increases focused section value by 1         |
+|                                     <kbd class="key">Arrow Down</kbd> | Decreases focused section value section by 1 |
+|                                        <kbd class="key">Page Up</kbd> | Increases focused section value section by 5 |
+|                                      <kbd class="key">Page Down</kbd> | Decreases focused section value section by 5 |
+|                                           <kbd class="key">Home</kbd> | Sets focused section to the minimal value    |
+|                                            <kbd class="key">End</kbd> | Sets focused section to the maximal value    |
+
+### Date Calendar
+
+All the available [views](/x/react-date-pickers/date-calendar/#views) implement the same basic keyboard support:
+
+|                                                                                                                                          Keys | Description                            |
+| --------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------- |
+|                                                                                                                    <kbd class="key">Tab</kbd> | Moves focus to the next element        |
+|                                                                                       <kbd class="key">Shift</kbd>+<kbd class="key">Tab</kbd> | Moves focus to the previous element    |
+| <kbd class="key">Arrow Up</kbd>, <kbd class="key">Arrow Down</kbd>,<br> <kbd class="key">Arrow Left</kbd>, <kbd class="key">Arrow Right</kbd> | Moves focus among the available values |
+
+In addition to the basic keyboard support above, the day view component implements its own keyboard interactions:
+
+|                             Keys | Description                                                       |
+| -------------------------------: | :---------------------------------------------------------------- |
+|   <kbd class="key">Page Up</kbd> | Moves calendar to next month, keeping focus on the same day       |
+| <kbd class="key">Page Down</kbd> | Moves calendar to previous month, keeping focus on the same day   |
+|      <kbd class="key">Home</kbd> | Moves focus to the first day of the week within the current month |
+|       <kbd class="key">End</kbd> | Moves focus to the last day of the week within the current month  |
```

Comment:
Are you sure about the `current month` part here?
I'm pretty sure that it applies to `DateRangeCalendar` instead. 🙈